docs/concepts/redirects.mdx

```diff
@@ -40,105 +38,125 @@ will fail, however, for https://anotherapp.com will succeed on any path.
-the user to your application home screen after a logout or to a specific url on
-a sub-domain after a settings password update. On project creation the redirect
-flows are set to the default Ory Cloud Managed UI. For more information on the
-Ory Managed UI, please see the [Managed User Interface](./managed-ui.mdx)
-documentation.
-
-### Login, Registration and Settings
-
-The Login and Registration flows have three fields (all optional):
-
-1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-OIDC Redirect URL` (**not supported**)
-
-And Settings has three fields (all optional):
+the user to your application home screen after a logout or to a specific URL on
+a sub-domain after a settings password update.
 
-1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-Profile Redirect URL`
+On project creation the redirect flows are set to the default
+[Managed User Interface](./managed-ui.mdx).
 
-:::info
+### Login, Registration, and Settings
 
-Setting `Post-OIDC` redirects for Login and Registration will be supported in a
-future release. For more information see the
-[Ory Cloud Roadmap](../cloud/cloud-roadmap.mdx).
-
-:::
+The Login and Registration flows have three fields (all optional):
 
-When setting the `Default Redirect URL` as an FQDN (fully qualified domain name)
-such as `https://domain.example/login` it will overwrite the
-`Global Redirect URL`. As a path it will replace the `Global Redirect URL` path,
-for example a path such as `/login` will alter
-`https://domain.example/global/path` to `https://domain.example/login`.
+1. Default Redirect URL domain or path relative to your application URL
+2. Post-Password Redirect URL
+3. Post-OIDC Redirect URL
+
+The Settings flow has three fields (all optional):
+
+1. Default Redirect URL domain or path relative to your application URL
+2. Post-Password Redirect URL
+3. Post-Profile Redirect URL
+
+To overwrite the Global Redirect URL set the Default Redirect URL to a fully
+qualified domain name
+([FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) - for example
+`https://domain.example/login`).  
+To replace the Global Redirect URL path set the Default Redirect URL to a
+path. For example when setting it to `/login`,
+`https://domain.example/global/path` becomes `https://domain.example/login`.
+
+The Default Redirect URL is used when Ory isn't sure where to redirect. It’s a
+good idea for this to be your default app URL, for example `/` or the dashboard.
+Use sub-conditions to control where users are redirected after registration,
+login, and other self-service flows.
+
+On sub-conditions such as After Password or After Profile an FQDN or a path
+can be set.  
+Set the Post- redirect to a FQDN to replace the closest relative, starting
+with the Login/Registration Default Redirect URL and then the
+Global Redirect URL.  
```

Comment:
I think we should call just call this `Default Redirect URL` and not introduce `Global`, I think they refer to the same thing?

Reply:
I kind of went with the wording in the original doc - which follows the names in the UI.

However I think it makes sense to omit the Global and call it the Default Redirect URL.

<img width="762" alt="Screenshot 2022-03-18 at 17 13 19" src="https://user-images.githubusercontent.com/55394283/159040821-f0717237-ed4b-47b5-a7ff-654752b7b968.png">

"Global Redirect URL" would be the "global default redirect URL" in the very top 

"Default Redirect URL" means the "Post- Default Redirect URLs", so for example "Post-Login Default Redirect URL".

That is the source of confusion. It is probably sensible to give the Post-X flows a common name to avoid this confusion? Maybe "Default Redirect URLs for Self-Service Flows"?
and then the Default Redirect URL is the global one.

Reply:
How about "Redirect target after successful <login|registration|...>" and "Redirect target after successful <login|registration|...> with <password|social|...>"?